### DISCUSSION TITLE
Node.js version: 12 to 16

### DISCUSSION THREAD

**Discussion by mijosephr** [JamesIves/github-pages-deploy-action#1240] [Ideas]
Hello @JamesIves and thank you for this very useful workflow!

The Node.js version used on the `main` branch is 12 while it is 16 on the `dev` branch.

My company only allows workflows that run on Node.js version 16. Would you mind doing a new release or do you know when the next one will be?

Thanks a lot!

**Comment by JamesIves** [comment]
This has been updated as of v4.4.1 - https://github.com/JamesIves/github-pages-deploy-action/releases/tag/v4.4.1
